p2panda-rs/src/schema/validate/blob.rs

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: AGPL-&3.0-or-later
+
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+use crate::operation::plain::{PlainFields, PlainValue};
+use crate::schema::validate::error::BlobError;
+
+/// @TODO: Do we want a max and what should it be?
+const MAX_BLOB_LENGTH: i64 = 100000; // 1GB
```

Comment:
This probably also relates to this restriction here, right? https://github.com/p2panda/aquadoggo/compare/development#diff-42ddd3729325393c2a3a0f2ae68495cfaca656de0ec6b29a171669fbbce26b30R19

If we set the max. pieces number to 10.000 then we gonna have a maximum length of 10.000 * 256 bytes = 2.56 Megabyte.

Probably we want to adjust these values on both ends and make them reflect the same maximum.

Reply:
Yeh, `MAX_BLOB_LENGTH` relates to the size of the complete blob file in bytes (as stated in the `length` field), and then on the other end we have `MAX_BLOB_PIECES` which relates to the maximum number of pieces a blob can contain. 

Reflecting on it though, I don't think we need a limit here in `p2panda-rs`. It's something we configure in `aquadoggo`.